src/mnode/mnode-consts.h

```diff
@@ -26,6 +27,7 @@ enum class TicketID : uint8_t {
     Buy,          // Art buy ticket
     Trade,        // Art trade ticket
     Down,
+    Username,
```

Comment:
We have to be very carefull with this enum. Each ticket has its unique ID, which should not change in any case.
I think we even need to assign specific values to all ticket IDs here.
There is another PR that adds new ticket id before existing take down ticket (not sure if we have already any takedown tickets in mainnet).
Ticket ID is written to blockchain, and we can't change IDs.

Reply:
Discussed with team and baselined the enum will be:
enum class TicketID : uint8_t {
    PastelID = 0, // Pastel ID registration ticket
    Art,          // Art registration ticket
    Activate,     // Art activation ticket
    Sell,         // Art sell ticket
    Buy,          // Art buy ticket
    Trade,        // Art trade ticket
    Down,
    Royalty,
    Username,
    COUNT         // number of ticket types
};

Reply:
fixed